### ISSUE TITLE
Set target hotkey bug

### ISSUE THREAD

**Issue by 6AKU66** [beyond-all-reason/Beyond-All-Reason#559]
Set target hotkey (Y key) behave differently unlike UI button.
You can't attack ground using hotkey, but UI button allow that without problem.

https://youtu.be/RHWZts8_mww

**Comment by WatchTheFort** [comment]
A Set Target order should not be able to be given on empty ground. The button will need to be fixed.

**Comment by 6AKU66** [comment]
Though probably there's some scenarios when it's can useful for someone (attack on empty ground). Maybe made it tweakable?

**Comment by WatchTheFort** [comment]
An attack order on empty ground already accomplishes that

**Comment by 6AKU66** [comment]
But isn't unit stop when do attack order on empty ground?

**Comment by WatchTheFort** [comment]
No, a unit will continuously attack when given an attack order targeted on empty ground.

**Comment by Zecrus2** [comment]
Set target on empty ground is useful for tremors, catapults, dominators, etc.  It lets you keep firing at the area while moving your units.

**Comment by 6AKU66** [comment]
Probably you got me wrong. I meant that if unit was walking and attack order was given unit just stop walking, if i use set target unit not stop walking.
Maybe change attack order behaviour? So if unit was walking it's won't stop walking if attack order will be given...

**Comment by Goopy16204** [comment]
You can use alt+y to set target on the ground, this will keep the unit firing even after a move order is made. Be careful though because if that target is still there when your units get in range, even if they are being shot by something else they will target the ground. Cancel target is j iirc

**Comment by 6AKU66** [comment]
Thank you. I didn't manage find hotkey alt+y on website.
